mongodbatlas/data_lakes_test.go

```diff
@@ -0,0 +1,412 @@
+package mongodbatlas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/openlyinc/pointy"
```

Comment:
this file is not go imported, I think our `make fmt` may be missing that, travis is failing on this